fabtests/component/dmabuf-rdma/rocr.c

```diff
@@ -0,0 +1,171 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <unistd.h>
+#include <string.h>
+#include "util.h"
+#include "rocr.h"
+
+/*
+ * Memory allocation & copy routines using oneAPI L0
+ */
+
+#define MAX_GPUS	(8)
```

Comment:
can you please set it at least to 16 to handle Hayabusa nodes?

Reply:
actually, we should probably delete this here, since there is also the same define statement in rocr.h